docs/BreakingChanges/114 GlyphRun.ComputeInkBoundingBox() and FormattedText.Extent return different values beginning in .NET 4.5.md

```diff
@@ -0,0 +1,37 @@
+## 114: GlyphRun.ComputeInkBoundingBox() and FormattedText.Extent return different values beginning in .NET 4.5
```

Comment:
{
  "Id": "114",
  "Title": "GlyphRun.ComputeInkBoundingBox() and FormattedText.Extent return different values beginning in .NET 4.5",
  "Details": "Improvements were made to GlyphRun.ComputeInkBoundingBox() and FormattedText.Extent in the .NET Framework 4.5 to address issues where the boxes were too small for the contained glyphs in some cases in the .NET Framework 4.0. As a result of this, some bounding boxes will be larger beginning in the .NET Framework 4.5, resulting in subtle differences in UI layout.",
  "Suggestion": "Be aware that some glyph bounding box sizes have increased. These changes will usually improve presentation and hit box testing, but if the older (pre-.NET 4.5) behavior is desired, it can be opted into by adding the following entry to the app.config file:\n\n```xml\n<appsettings>\n<add key=\"IncludeAllInkInBoundingBox\" value=\"false\">\n</appsettings>\n```",
  "Link": "http://support.microsoft.com/en-us/kb/3009678",
  "Markdown": null,
  "ApplicableApis": [
    "M:System.Windows.Media.GlyphRun.ComputeInkBoundingBox",
    "M:System.Windows.Media.FormattedText.get_Extent",
    "P:System.Windows.Media.FormattedText.Extent"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": null,
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/971148",
  "Notes": "Source analyzer status: Planned - No tracking work item yet",
  "IsRetargeting": false,
  "ImpactScope": 3
}

**Parsed APIs:**
- [x] M:System.Windows.Media.GlyphRun.ComputeInkBoundingBox
- [x] M:System.Windows.Media.FormattedText.get_Extent
- [x] P:System.Windows.Media.FormattedText.Extent